articles/azure-functions/functions-reference-python.md

```diff
@@ -637,7 +637,7 @@ Make sure that you also update your function.json to support the OPTIONS HTTP me
     ...
 ```
 
-This method is used by the Chrome browser to negotiate the allowed origins list. 
+This HTTP method is used by web browsers to negotiate the allowed origins list. 
```

Comment:
Using the OPTIONS HTTP method to do CORS negotiation is not specific to Chrome. All browsers do it.